client/coral-framework/actions/auth.js

```diff
@@ -315,6 +316,9 @@ export const checkLogin = () => (dispatch) => {
     .catch((error) => {
       console.error(error);
       dispatch(checkLoginFailure(`${error.translation_key}`));
+    })
+    .then(() => {
+      pym.sendMessage('checkLogin', JSON.stringify(getState().auth));
```

Comment:
Why not passing the result of the rest call? The redux state of `auth` currently also contains state of the frontend interface related to auth, which I don't think we should expose.